shards/common.edn

```diff
@@ -9,11 +9,11 @@
 ;; Index of the Upload Function in Protos Pallet
 (def upload-index 0)
 ;; Index of the Patch Function in Protos Pallet
-(def patch-index 1)
+(def patch-index 2)
```

Comment:
No no no, put ban at end. We need to add this as an update as I'm resetting testnet already

Reply:
In general always add new functions at end of pallets cc @alebaffa